admin/extensions.php

```diff
@@ -1,165 +1,194 @@
 <?php
-
 namespace HM\BackUpWordPress;
-
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```

Comment:
I assume you're including this because we need access to one of the functions it contains? If so, I think it's best practise to include it just before you use the function and add a docblock comment explaining what it's being included for.

That way future developers will know why it's here and whether it's safe to remove or not.